composer.json

```
@@ -16,33 +16,33 @@
         "ext-PDO": "*",
         "ext-SPL": "*",
         "ext-xsl": "*",
-        "zetacomponents/mail": "~1.8",
-        "symfony/symfony": "~2.7.38 | ~2.8.31 | ^3.3.13",
-        "symfony-cmf/routing": "~1.1",
+        "zetacomponents/mail": "^1.8.3",
+        "symfony/symfony": "~2.8.46 | ^3.4.17",
+        "symfony-cmf/routing": "~1.4",
         "qafoo/rmf": "1.0.*",
-        "kriswallsmith/buzz": ">=0.9",
-        "tedivm/stash-bundle": "~0.6.1",
-        "sensio/distribution-bundle": "^3.0.36|^4.0.6|^5.0",
-        "nelmio/cors-bundle": "^1.3.3",
-        "hautelook/templated-uri-bundle": "~1.0 | ~2.0",
-        "pagerfanta/pagerfanta": "~1.0",
+        "kriswallsmith/buzz": "~0.16",
```

Actually:
```suggestion
        "kriswallsmith/buzz": "~0.16.0",
```
Otherwise we'll end up with 0.17 which introduced BC breaks (we support it since 7.x).

---

> I know, but as earlier requriment (>=0.9) allowed it I assumed we had fixed it here also

It works by accident, because tests using Buzz are executed from meta. `1.x` meta runs on Sf 2.8, but Buzz `0.17.x` requires Sf `3.4+` :)